features/reisdocumenten.feature

```diff
@@ -1,51 +1,92 @@
 # language: nl
 
-Functionaliteit: Het reisdocument van een ingeschreven persoon kan worden geraadpleegd op basis van het reisdocumentnummer
+Functionaliteit: Raadplegen van reisdocumenten op basis van een reisdocumentnummer
   Met het reisdocumentnummer als sleutel kan een reisdocument van een ingeschreven persoon worden geraadpleegd.
```

Comment:
Met het reisdocumentnummer kan worden gecontroleerd of het document in omloop is, en als identificatiemiddel kan worden gebruikt.

Reply:
Is aangepast